packages/platforms/inventec/x86-64/x86-64-inventec-d6254qs/platform-config/r0/src/python/x86_64_inventec_d6254qs_r0/__init__.py

```diff
@@ -0,0 +1,15 @@
+from onl.platform.base import *
+from onl.platform.inventec import *
+
+class OnlPlatform_x86_64_inventec_d6254qs_r0(OnlPlatformInventec,
+                                              OnlPlatformPortConfig_32x100):
+    PLATFORM='x86-64-inventec-d6254qs-r0'
+    MODEL="X86-D6254QS"
+    SYS_OBJECT_ID=".1.32"
```

Comment:
I believe this is in conflict with both the 7032 and 7054 platforms. 

Each platform must have its own unique SYS_OBJECT_ID. 

I suggest setting the 7032Q28B platform to 7032.28.32
I suggest setting the 7054Q28B platform to 7054.28.32
I suggest setting the 6254QS     platform to 6254.32

The values are up to you but they must be unique.